controller/articles.controller.js

```diff
@@ -60,3 +60,20 @@ exports.postComment = (req, res, next) => {
     })
 
 }
+
+exports.patchArticle = (req, res, next) => {
+    const { article_id } = req.params
+    const { inc_votes } = req.body
+  
+    patchArticleVoteById(article_id, inc_votes)
+        .then((article) => {
+            res.status(200)
+            .send(article)
```

Comment:
You should return the data on an appropriate key eg {article} 👍

Reply:
done :)